<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDlg</class>
 <widget class="QDialog" name="MainDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1145</width>
    <height>774</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QWidget" name="wdg_show" native="true">
       <property name="minimumSize">
        <size>
         <width>800</width>
         <height>600</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>999999</width>
         <height>999999</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="MyOpenGLWidget" name="wdg_showImg" native="true">
          <property name="minimumSize">
           <size>
            <width>800</width>
            <height>600</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(0, 0, 0);</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="wdg_info" native="true">
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="lb_mw">
          <property name="minimumSize">
           <size>
            <width>219</width>
            <height>79</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>219</width>
            <height>79</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_mh">
          <property name="minimumSize">
           <size>
            <width>219</width>
            <height>79</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>219</width>
            <height>79</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_imgw">
          <property name="minimumSize">
           <size>
            <width>219</width>
            <height>79</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>219</width>
            <height>79</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_imgh">
          <property name="minimumSize">
           <size>
            <width>219</width>
            <height>79</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>219</width>
            <height>79</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>微软雅黑</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_videoName">
          <property name="minimumSize">
           <size>
            <width>219</width>
            <height>79</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>219</width>
            <height>79</height>
           </size>
          </property>
          <property name="text">
           <string>video name</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QWidget" name="wdg_pross" native="true">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>77</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="lb_curtime">
          <property name="minimumSize">
           <size>
            <width>91</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>91</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string>00:00:00</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="slider_progress">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>22</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>22</height>
           </size>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_alltime">
          <property name="minimumSize">
           <size>
            <width>91</width>
            <height>31</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>91</width>
            <height>31</height>
           </size>
          </property>
          <property name="text">
           <string>00:00:00</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="wdg_bottom" native="true">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>76</height>
        </size>
       </property>
       <layout class="QHBoxLayout" name="layout_pbs">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="pb_open">
          <property name="minimumSize">
           <size>
            <width>61</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>61</width>
            <height>51</height>
           </size>
          </property>
          <property name="text">
           <string>本地</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_play">
          <property name="minimumSize">
           <size>
            <width>61</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>61</width>
            <height>51</height>
           </size>
          </property>
          <property name="text">
           <string>播放</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_pause">
          <property name="minimumSize">
           <size>
            <width>61</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>61</width>
            <height>51</height>
           </size>
          </property>
          <property name="text">
           <string>暂停</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_again">
          <property name="minimumSize">
           <size>
            <width>61</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>61</width>
            <height>51</height>
           </size>
          </property>
          <property name="text">
           <string>重播</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_stop">
          <property name="minimumSize">
           <size>
            <width>61</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>61</width>
            <height>51</height>
           </size>
          </property>
          <property name="text">
           <string>停止</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_last">
          <property name="minimumSize">
           <size>
            <width>61</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>61</width>
            <height>51</height>
           </size>
          </property>
          <property name="text">
           <string>上一个</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_next">
          <property name="minimumSize">
           <size>
            <width>61</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>61</width>
            <height>51</height>
           </size>
          </property>
          <property name="text">
           <string>下一个</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_speed">
          <property name="minimumSize">
           <size>
            <width>61</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>61</width>
            <height>51</height>
           </size>
          </property>
          <property name="text">
           <string>倍速</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_url">
          <property name="minimumSize">
           <size>
            <width>61</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>61</width>
            <height>51</height>
           </size>
          </property>
          <property name="text">
           <string>URL</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_quailty">
          <property name="minimumSize">
           <size>
            <width>61</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>61</width>
            <height>51</height>
           </size>
          </property>
          <property name="text">
           <string>画质</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_voice">
          <property name="minimumSize">
           <size>
            <width>61</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>61</width>
            <height>51</height>
           </size>
          </property>
          <property name="text">
           <string>音量</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_table">
          <property name="minimumSize">
           <size>
            <width>61</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>61</width>
            <height>51</height>
           </size>
          </property>
          <property name="text">
           <string>列表</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_set">
          <property name="minimumSize">
           <size>
            <width>61</width>
            <height>51</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>61</width>
            <height>51</height>
           </size>
          </property>
          <property name="text">
           <string>设置</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyOpenGLWidget</class>
   <extends>QWidget</extends>
   <header location="global">myopenglwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
